### PR TITLE
Update walkthrough.md

### PR DESCRIPTION
**Pull request by marktani**
feedback for platform walkthrough

### CODE REVIEW

File: content/guides/walkthrough.md [hunk 1]
```diff
@@ -1,71 +1,77 @@
...
+## Querying and Mutating your Nodes
+The playground gives you a GraphiQL editor to easily query and mutate your nodes. If you don't know what to enter, you can always rely on the docs or try your luck with the autocomplete function. If you already understand what GraphQL is, it is notable to tell you that we have a Simple API and a Relay API. You can find the endpoints on the top right corner of the window. You can also configure the GraphiQL editor to use the Simple or the Relay endpoint by selecting it in the dropdown menu location in the top center of the playground.
 
-If you don’t know how to query or mutate data using GraphQL, I highly encourage you to read through [this guide](www.learngraphql.com).
+If you don't know how to query or mutate data using GraphQL, I highly encourage you to read through [this guide](www.learngraphql.com).
```
Comment (marktani) [inline content/guides/walkthrough.md]: Should we also refer to the Simple/Relay API reference?

Not sure if it's a good place/time to link them.

Comment (brene) [inline content/guides/walkthrough.md]: Good point. I want to have @schickling 's insight on this. I think it's a good idea to mention it in two sentences or less.

Comment (schickling) [inline content/guides/walkthrough.md]: Yup, very good idea to cross reference from here!

**Comment by brene** [comment]
What do you guys think of this change?

**Comment by marktani** [comment]
@brene please resolve merge conflicts and merge so we can move along :)

**Comment by schickling** [comment]
I think we can close this in favor of the new guides structure

**Pull request closed without merging**